### DISCUSSION TITLE
subdomain opening coder instead of code-server app

### DISCUSSION THREAD

**Discussion by VicFleur** [coder/code-server#6882] [Q&A] [answered]
*my setup*
- Coder installed as a Docker container managed with Coolify
- Coder is available at coder.(server-name).com
- *.coder.(server-name).com goes to Cloudflare and has its own certificate
- code-server starts a Nuxt 3 app on port ```3500```
- code-server is set to start with ```--proxy-domain coder.(server-name).com```
- Coolify is set to automatically route via Traefik both coder.(server-name).com and 3500.coder.(server-name).com to the Coder instance

*expected result*
at this point, I would expect to open 3500.coder.(server-name).com and find the Nuxt app

*actual result*
instead, I’m present with Coder’s login screen, and when I login I’m simply redirected to the /workspaces page

I want to use the subdomain proxy to avoid all issues with paths, what am I doing wrong?

**Comment by code-asher** [comment, marked as answer]
Coder does not forward `<port>.<domain>` to a workspace, as far as I know, so code-server will never see the request.  `--proxy-domain` is a code-server thing only and Coder has no awareness of it.

Coder has its own proxy, though.  If you set it up then you should be able to access a URL like `https://<port>--<agent>--
<workspacename>--<user>--apps.dev.coder.com`.  From the dashboard, if you click into a workspace then in the upper right of the card you should see "Open ports" and clicking on that should let you access any port (so you do not have to type the URL manually).

Also, code-server should integrate with Coder's proxy automatically so if you have it set up, it should automatically be used in the ports panel of code-server as well.

https://coder.com/docs/networking/port-forwarding#dashboard

**Reply by VicFleur** [reply]
Yes! That worked perfectly 🎉
